engines/wintermute/base/gfx/opengl/light3d.cpp

```diff
@@ -123,9 +123,9 @@ bool Light3D::loadFrom3DS(Common::MemoryReadStream &fileStream) {
 			byte g = fileStream.readByte();
 			byte b = fileStream.readByte();
 
-			_diffuseColor = r;
+			_diffuseColor = r << 16;
```

Comment:
Doubt so. Now that being said, this is not the only critical place anyways. At some point, probably in a week or so, I will go trough all these places, try to ensure endian-safety and then test it. Actually, there was someone one the ResidualVM forum, who essentially volunteered to do such tests on a PowerPC Mac. Otherwise I have to setup a VM, I guess.